docs/references.md

```diff
@@ -0,0 +1,8 @@
+## References
+
+* *Scopatz, Anthony, and Kathryn D. Huff*. 2015. **Effective Computation in Physics. 1 edition.** S.l.: O’Reilly Media.
```

Comment:
Can we make this a link to the page to buy the book? http://shop.oreilly.com/product/0636920033424.do